src/Symfony/Component/Security/Core/Authentication/Token/AbstractToken.php

```diff
@@ -261,7 +261,7 @@ private function hasUserChanged(UserInterface $user)
         }
 
         if ($this->user instanceof AdvancedUserInterface && $user instanceof AdvancedUserInterface) {
-            @trigger_error(sprintf('Checking for the AdvancedUserInterface in %s has been deprecated in 4.1. Implement the %s to check if the user has been changed,', __METHOD__, EquatableInterface::class), E_USER_DEPRECATED);
+            @trigger_error(sprintf('Checking for the AdvancedUserInterface in "%s()" has been deprecated in 4.1. Implement the %s to check if the user has been changed,', __METHOD__, EquatableInterface::class), E_USER_DEPRECATED);
```

Comment:
should we change it to `is deprecated since Symfony 4.1` for consistency? same below

Reply:
good catch, changed it